continuous-integration/linux/install-sdist.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Same with this filename, could we use longer names? It might make it easier for newer people to contribute

Reply:
Renaming to "build-and-install-source-distribution.sh", since it actually does a complete build based on the sdist.